buzzmobile/tests/sense/test_inputer.py

```diff
@@ -1,55 +1,21 @@
-import rostest
 import unittest
-import rospy
-import time
 from std_msgs.msg import String
-import roslaunch
+from tests import rostest_utils
 
-PKG = 'buzzmobile'
 NAME = 'test_inputer'
 
 
+@rostest_utils.with_roscore
 class TestInputer(unittest.TestCase):
 
-    def run_node_with_callback(self, callback):
-        rospy.init_node(NAME, anonymous=True)
-
-        rospy.Subscriber('destination', String, callback)
-        timeout = time.time() + 2.0
-
-        while time.time() < timeout and not rospy.is_shutdown():
-            time.sleep(0.1)
-
-    def __init__(self, *args):
-        super().__init__(*args)
-
-
     def setUp(self):
         self.success = False
-        node = roslaunch.core.Node('buzzmobile', 'inputer.py')
-        launch = roslaunch.scriptapi.ROSLaunch()
-        launch.start()
-        self.process = launch.launch(node)
-
-    def test_inputer_no_input(self):
-        def cb(data):
-            self.success = (data.data == '')
-
-        self.run_node_with_callback(cb)
-        assert self.success
 
+    @rostest_utils.launch_node('buzzmobile', 'inputer.py', 'params.launch')
```

Comment:
Is it a good idea to have `launch_node` manage starting launch files too? In my head we'd have a decorator for each. But I can see that maybe things won't be super guaranteed to run in order so idk.